tests/sql/test_orderbys.py

```diff
@@ -0,0 +1,98 @@
+import daft
+
+df = daft.from_pydict(
+    {
+        "text": ["g1", "g1", "g2", "g3", "g3", "g1"],
+        "n": [1, 2, 3, 3, 4, 100],
+    }
+)
+
+
+def test_orderby_basic():
+    df = daft.sql("""
+        SELECT * from df order by n
+    """)
+
+    assert df.collect().to_pydict() == {
```

Comment:
Also btw, `.collect()` here is not necessary because `.to_pydict()` includes materialization too.